scripts/voice_activity_detection/vad_tune_threshold.py

```diff
@@ -19,11 +19,34 @@
 from nemo.collections.asr.parts.utils.vad_utils import vad_tune_threshold_on_dev
 from nemo.utils import logging
 
+"""
+This script is designed for thresholds tuning for postprocessing of VAD
```

Comment:
does this script return threshold or onset, offset as well? or any other parameters?

Reply:
yes this will return all params to be tuned given by the command line.
If only input onset and offset, min_duration_on, then it will return {"onset":xx, "offset":xx, "min_duration_on": xx}